aws/aws_instance.go

```diff
@@ -273,18 +273,18 @@ func (p *AWS) CreateInstance(ctx *lepton.Context) error {
 	}
 
 	// Specify the details of the instance that you want to create.
-	ctx.Logger().Debug("running instance with input %v", instanceInput)
+	ctx.Logger().Debugf("running instance with input %v", instanceInput)
 	_, err = svc.RunInstances(instanceInput)
 	if err != nil {
-		log.Errorf("Could not create instance", err)
+		log.Errorf("Could not create instance %v", err)
 		return err
 	}
 
 	log.Info("Created instance", tagInstanceName)
 
 	// create dns zones/records to associate DNS record to instance IP
 	if cloudConfig.DomainName != "" {
-		ctx.Logger().Debug("associating domain to the created instance")
+		ctx.Logger().Debugf("associating domain to the created instance")
```

Comment:
not formatted?

Reply:
Fixed by https://github.com/nanovms/ops/pull/1102/commits/2920037fc326cf982510341c6c127f312569fa73